cltk/corpus/arabic/alphabet.py

```diff
@@ -0,0 +1,16 @@
+"""Arabic alphabet"""
+
+
+#Arabic digits from 0 to 9
+DIGITS = ['۰', '۱', '۲', '۳', '٤', '۵', '٦', '۷', '۸', '۹']
+
+#Diacritics
+SHORT_VOWELS = ['َ', 'ِ', 'ُ']
```

Comment:
I was going to add Shadda (the w shaped one), but I had no knowledge of the rest, so thank you. Could you add them please?